helpers/helpers.html

```diff
@@ -85,19 +85,20 @@
         var deltaScrollTop = scrollTop - currentScrollTop;
         var deltaScrollLeft = scrollLeft - currentScrollLeft;
         var duration = 300;
-
-        (function updateFrame() {
+        var updateFrame = (function updateFrame() {
```

Comment:
Don't need the function name in the function declaration (e.g. `function() {...`). I don't know if this actually breaks anything, but it's certainly weird to have two places that declare `updateFrame`.